### DISCUSSION TITLE
1.25.1

### DISCUSSION THREAD

**Discussion by Koenkk** [Koenkk/zigbee2mqtt#12337] [General]
# Improvements
- [#4061](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4061) Expose `action` and `operation_mode` for Niko 552-721X2 and 552-721X1
- [#4055](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4055) Improve electrical measurement reporting for Datek HSE2905E
- [#4080](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4080) Expose battery for PLAID SYSTEMS PS-SPRZMS-SLP3
- [#4084](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4084) Expose additional functionality for Neo NAS-PD07 and Fantem ZB006-X
- [#4060](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4060) Expose acceleration for Smartthings 3321-S
- [#4095](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4095) Expose additional actions for Fantem ZB006-X
- #3949 Expose water_leak for IKEA E1842
- [#4114](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4114) Expose voltage and energy for Xiaomi SP-EUC01
- [#4123](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4123) Prefer hue/saturation color mode over X/Y for Lidl HG08633 
- #12146 Mark `calibration`, `calibration_time` and `motor_reversal` as config sensors in Home Assistant
- [#4125](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4125) Support power on behavior for Sonoff ZBMINI-L
- #12204 Expose battery % for HEIMAN SMHM-I1
- [#4146](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4146) Expose additional functionality for Woox R7049
- [#4152](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4152) Add `child_lock` and `led_enable` support to the Niko Outlets
- [#4154](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4154) Expose additional functionality for Namron 4512737/4512738
- [#4102](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4102) Expose additional functionality for Fantem ZB006-X
- [#4173](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4173) Expose `temperature_scale` for Neo NAS-PD07 and correct humidity/temperature min/max ranges for Neo NAS-AB02B0 and NAS-PD07
- [#4182](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4182) Expose `child_lock` for Moes ZP-LZ-FR2U
- [#4180](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4180) Expose new features and fixes for Fantem ZB003-X

# Fixes
- #12022 Fix Spotmau SP-PS1-02 not working
- #12025 Fix curtain motor state incorrect in Home Assistant
- #12023 Fix zigbee model for 005f0c3b TuYa U86KCJ-ZP
- #12038 Fix invalid Home Assistant device class for garage_door_contact
- #11033 #12019 Fix incorrect co2/pm25 values for TuYa TS0601_air_quality_sensor
- [#4066](https://github.com/Koenkk/zigbee-herdsman-converters/issues/4066) Fix incorrect action for TuYa U86KCJ-ZP
- #12076 Remove tamper from HEIMAN SMHM-I1 (not supported)
- #12077 Remove battery_low and tamper from Feibit SGA01ZB (not supported). 
- [#4086](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4086) Fix configure failing for some electrical measurement devices due to unsupported `acFrequencyDivisor`/`acFrequencyMultiplier` attribute
- #12080 Fix `Cannot set property 'softwareBuildID' of undefined` exception when pairing Develco/Frient SMSZB-120
- #12097 Fix no power measurements reported for TS011F plugs with appVersion 74
- #12018 Fix incorrect `water_leak` state for Xiaomi SJCGQ11LM
- [#4085](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4085) Fix no energy reported for Micro Matic Norge AS ZB250
- #12149 Fix Moes ZP-LZ-FR2U sockets controlled together
- #12154 Fix Xiaomi XDD13LM, XDD12LM and XDD11LM state incorrect when brightness is set to 1
- [#4129](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4129) Fix LELLKI XF-EU-S100-1-M not controllable
- [#4130](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4130) Fix duplication of voltage value for Xiaomi JTYJ-GD-01LM/BW
- #12034 Fix out of memory error when starting on system with low RAM
- [#4143](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4143) Fix position inverted for Zemismart ZM-AM02_cover (`_TZE200_gubdgai2`)
- #4561 Fix occasional flickering of iCasa lamps
- [#4160](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4160) Define `load_estimate` min/max value for Danfoss 014G2461
- #12281 Fix Eglo Fueva ceiling light mis-identified
- #11033 Fix TS0601_air_quality_sensor formaldehyd and voc units
- [#4163](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4163) Fix Xiaomi ZNJLBL01LM position sometimes incorrect
- #12276 Fix `No converter available for 'on_time'` for lights
- #12279 Fix Xiaomi Aqara S2 lock ZNMS12LM state being `ON`/`OFF` instead of `LOCK`/`UNLOCK`
- [#4171](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4171)  Add OTA support and fix battery for Xiaomi WSDCGQ12LM
- #11033 Fix invalid pm25 values for TuYa TS0601_air_quality_sensor
- #12225 Fix `No converter available` warning for Neo NAS-AB02B0 and NAS-PD07
- [#4181](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4181) Fix battery for Xiaomi WXKG16LM
- [#4180](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4180) Fix Fantem ZB006-X configure error
- #12313 Fix `Received unexpected Tuya DataPoint #2` error for `_TZE200_dfxkcots`

# New supported devices
This release adds support for **41** new devices:
- `14153806L` Lidl Livarno smart LED ceiling light
- `3315-L` Iris Water sensor
- `43023` VBLED ZigBee AC phase-cut dimmer
- `4500994` Connecte Smart thermostat
- `5060930P7` Philips Hue White & Color Ambiance Centris ceiling light (3 spots)
- `5061030P7` Philips Hue White & Color ambience Centris ceiling light (2 spots) (black)
- `5412748727371` Prolight E27 white and colour bulb
- `5412748727395` Prolight GU10 spot dimmable
- `9133` Led Trading Pushbutton transmitter module
- `915005996901` Philips Hue white ambiance ceiling light Enrave L with Bluetooth
- `915005997601` Philips Hue Devere M white ambiance white & dimmer
- `915005997701` Philips Hue Devere L white ambiance white & dimmer
- `915005998201` Philips Hue Bluetooth white & color ambiance ceiling lamp Infuse
- `929002294102` Philips Hue white candle bulb E14 bluetooth
- `929003046201` Philips Hue White Ambiance Runner triple spotlight
- `A9MEM1570` Schneider Electric PowerTag power sensor
- `BY 286 C` Innr B22 bulb RGBW
- `CCT592011` Schneider Electric Wiser water leakage sensor
- `CS-Z-CZ-2402` DNAKE Smart socket
- `DS20Z07B` ORVIBO Downlight (S series)
- `E2102` IKEA PRAKTLYSING cellular blind
- `E2103` IKEA TREDANSEN cellular blind
- `FB21-001` Lidl Livarno Lux switch and dimming light remote control
- `FUT089Z` MiBoxer RGB+CCT Remote
- `GDC311ZBQ1` TuYa LoraTap garage door opener with wireless sensor
- `HG08008` Lidl Livarno Home LED ceiling light
- `HG08633` Lidl Livarno gardenspot RGB
- `IA-CDZOTAAA007MA-MAN` ADEO ENKI LEXMAN E27 7.2 to 60W LED RGBW
- `IST-CDZFB2AS007NA-MZN-01` ADEO ENKI LEXMAN E27 LED white
- `JM-TRH-ZGB-V1` TuYa Temperature & humidity sensor with clock
- `JY-GZ-01AQ` Xiaomi Aqara smart smoke detector
- `LA-5KEY-RGBW` Ynoa 5 key control for RGBW light
- `LA-GU10-RGBW` Ynoa Smart LED GU10 RGB CCT
- `OGL 130 C` Innr Outdoor smart globe lights
- `RTCGQ14LM` Xiaomi Aqara P1 human body movement and illuminance sensor
- `SP 110` Innr Smart plug
- `SP2610ZB` Sinopé Zigbee smart outlet
- `SR-ZG9001K2-DIM` Sunricher Zigbee wall remote control for single color, 1 zone
- `TS0601_co2_sensor` TuYa NDIR co2 sensor
- `TS0601_smart_human_presense_sensor` TuYa Smart Human presence sensor
- `WNRR15/WNRR20` Legrand Outlet with power consumption monitoring

<hr /><em>This discussion was created from the release <a href='https://github.com/Koenkk/zigbee2mqtt/releases/tag/1.25.1'>1.25.1</a>.</em>

**Comment by andreas-git** [comment]
Hi,

after updating to 1.25.1 my Z2B frontend in HA seems not to work, just blank page.
My Z2M configuration.yaml:
`
homeassistant: true
mqtt:
  server: mqtt://core-mosquitto:1883
  user: addons
  password: waePaechovu1wahSheid3eiy2waemahQuaeceego0toxoghohdo3Wu6Iez3Aeg6i
  keepalive: 60
  reject_unauthorized: true
  version: 4
serial:
  port: tcp://192.168.178.23:8888
  adapter: ezsp
  baudrate: 115200
  rtscts: true
frontend:
  port: 8099
permit_join: true
devices:
  '0x847127fffedd1a9d':
    friendly_name: '0x847127fffedd1a9d'
  '0xcc86ecfffe951311':
    friendly_name: '0xcc86ecfffe951311'
  '0xcc86ecfffe9550ba':
    friendly_name: '0xcc86ecfffe9550ba'
  '0x00158d0002ffeaa3':
    friendly_name: 0x00158d0002ffeaa3 Tint Remote
    legacy: false
  '0x804b50fffe052b4e':
    friendly_name: '0x804b50fffe052b4e'
advanced:
  log_syslog:
    app_name: Zigbee2MQTT
    eol: /n
    host: localhost
    localhost: localhost
    path: /dev/log
    pid: process.pid
    port: 123
    protocol: tcp4
    type: '5424'
  channel: 15
  network_key: *****
groups: {}
`
Rolling back to 1.25.0-1 seems fine.

Any Ideas ?

Best Regards

**Reply by fiodorow** [reply]
Having the same issue, frontend is blank but apart from this z2m works OK.

**Reply by andreas-git** [reply]
> Having the same issue, frontend is blank but apart from this z2m works OK.

me too, z2m works, but no web front end.

**Reply by GuidoKJ** [reply]
I can confirm that I cannot longer access the frontend on 1.25.1.

**Reply by andreas-git** [reply]
Hi,

i created issue for that:
https://github.com/zigbee2mqtt/hassio-zigbee2mqtt/issues/333

**Reply by vandad51** [reply]
Having same issue.

**Reply by navycrow** [reply]
+1

edit: uninstall / install seems to resolve the problem

**Reply by zorruno** [reply]
+1 (previous version restored from backup)

**Reply by mrJudahBella** [reply]
Yup, same issue here. 

EDIT: Fixed by emptying caches on Safari. (Develop -> Empty Caches)
EDIT2: Resetting frontend cache on the HA iOS app fixes the problem also.

**Reply by jonaseymour** [reply]
+1 rolled back to previous version.

**Comment by billythunder88** [comment]
emptying the browser cache worked for me.

**Comment by bribeck** [comment]
I installed this update and got the new Aqara P1 motion sensor it works great but the blue detection light seems to be off by default is there a way to turn in on?

**Reply by laszlojakab** [reply]
For me it was turned on by default, I would like to turn it off :-) Could you change the blue detection light to on/off?

**Reply by lux73** [reply]
i don't think you got support for such an 4 month old Version like 1.25.1

try updating pls and then open new Thread for the detection Light "issue"

**Comment by kelvinfanSZ** [comment]
Nice, Schneider Electric PowerTag be supported! Who is the contributor?

Is a special NCP firmware is needed to support this device? I worked on this device in Schneider Electric for few months.

**Reply by Koenkk** [reply]
@filips added it
